tests/hash-playwright/tests/profile-page.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { sleep } from "@local/hash-isomorphic-utils/sleep";
+// eslint-disable-next-line no-restricted-imports
+import { test as testTolerateConsoleErrors } from "@playwright/test";
+
+import { loginUsingTempForm } from "./shared/login-using-temp-form";
+import { resetDb } from "./shared/reset-db";
+import { expect } from "./shared/runtime";
+
+testTolerateConsoleErrors.beforeEach(async () => {
+  await resetDb();
+});
+
+/**
+ * @todo H-2006 fix bugs on profile page and revert to using 'test' from ./shared/runtime
+ */
+testTolerateConsoleErrors("a user's profile page renders", async ({ page }) => {
+  await loginUsingTempForm({
+    page,
+    userEmail: "alice@example.com",
+    userPassword: "password",
+  });
+
+  await page.goto("/@alice");
+
+  await expect(page.locator("text=@alice")).toBeVisible();
+  await expect(page.locator('text="Profile"')).toBeVisible();
+
+  await page.click("text=Add a bio for Alice...");
+
+  await sleep(2_000);
+
+  await page.keyboard.type("Alice's bio");
+
+  await page.click("[aria-label='Save Bio']");
+
+  await sleep(2_000);
+
+  await page.reload();
+
+  await expect(page.locator("text=@alice")).toBeVisible();
```

Comment:
We could also check if the `Save Bio` persisted

Reply:
Done in https://github.com/hashintel/hash/pull/3933/commits/4889c986a7e85a9099af507313868f37532fab31